JavascriptMornings/Maitee
# Adding basic radio functions

## Review checklist


## Testing

Steps to test the change:
In index.js, first, call the function 'turnRadio', and then call 'getRadioMode' and 'setRadioMode' functions to see the different modes of the radio. For example, using the radio for the first time:
console.log('Turn on radio', Maitee.turnRadio());// Should return true (on)
Maitee.setRadioMode();// Should radioMode be incrementing by 1 (this part is depending the case)
console.log(Maitee.getRadioMode());// Shold return 'usb' mode

## Additional notes

Play with the functions and see the different cases. Other features will be added soon :)

## Deploy checklist
